packages/tasit-contracts/.gitignore

```diff
@@ -1 +1,2 @@
 build/**
+3rd-parties/**/scripts/config/default.js
```

Comment:
That file isn't ignored because it stores password, it's because `config` folder has one config per network and npm script creates a copy to `default.js` that is used on migration.
The ignoring is to avoiding duplicated file.

For now, I'm using same accounts for test and migrations (and mnemonic is stored on `migrate.sh` shell script). 
I hope that when we need to keep privKey secret we will already using the same `truffle` project  (w/ external compilers feature) and the passwords will be loaded from the main `truffle-config.js`, at that time, these scripts that I wrote inside of `3rd-parties` folder will be removed/refactored.

Reply:
Ah right, makes sense.

I like that suggested plan for the future.

Reply:
Feel free to resolve this. And create an issue for that plan you mentioned if you like.

Reply:
Issue already created: https://github.com/tasitlabs/TasitSDK/issues/247
Resolving this.

Reply:
Ah right. Thanks.